server/flyway/local_sql/V1001__add_bceid_delegated_admin_for_testing.sql

```diff
@@ -0,0 +1,71 @@
+-- create user for running test, test for the security check for delegated admin grant/remove access
```

Comment:
Added this new file under local_sql, because the Backend API db user has no write access to create delegated admin, I created the delegated admin privilege here, so when we run it locally or run test, will have a test user has some privilege we can test for.